### ISSUE TITLE
[Toasts] - circular gauge dom / accessibility issue

### ISSUE THREAD

**Issue by nsarradin** [LuccaSA/lucca-front#2300]
Two issues identified on the circular gauge in `toast.component.html` :
- `div.circularGauge` should be replaced by a `span` as buttons can't include `div`
- `aria-hidden="true"` should be added on the svg of the gauge

**Comment by jeremie-lucca** [comment]
Circular gauge was removed :)
